main.go

```diff
@@ -20,6 +21,7 @@ func main() {
 
 	//routers
 	product.ProductRouter(ginEngine)
+	home.GetHome(ginEngine)
```

Comment:
Please stick with  the structure of the codebase for initiating routers

Reply:
Noted